bump-versions.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
```

Comment:
One thing I'm a bit proud of is that there aren't many files/directories in the root of this repository. Consider moving this (alongside `runtime-benchmark.sh`, which might need adjustments after changing its location) to `tools` or `scripts` directory or something along those lines.

Reply:
Sure, that makes sense to make the repository easier to use.

I fixed this script, it already automatically changes to the root of the repository, so the only required change is:
```sh
cd "$(dirname "$0")/.."
```

I'll do the other script in its existing PR.